src/pkgdev/scripts/pkgdev_mask.py

```diff
@@ -99,6 +114,18 @@ def __str__(self):
         lines.extend(map(str, self.atoms))
         return '\n'.join(lines)
 
+    def send_email(self):
```

Comment:
I think I'd make this a standalone function not under the Mask class and have it accept a string arg or the mask obj depending on how generic you feel it should be.